File: web/js/kits.js
kits.init_table1 = function (table) {
    table.jqGrid({
        datatype: "local",
        rownumbers: true,
        colNames: ['id', 'Артикул', 'Название', 'Основная',
            'Внутренняя', 'Внешняя', 'Ширина', 'Длина', 'Кол-во', 'Угол1', 'Угол2'],
        colModel: [
            {name: 'id', hidden: true},
            {name: 'artikl_id', width: 80, sorttype: "text"},
            {name: 'artikl_id', width: 200, sorttype: "text"},
            {name: 'color1_id', width: 80, sorttype: "text"},
            {name: 'color2_id', width: 80, sorttype: "text"},
            {name: 'color3_id', width: 80, sorttype: "text"},
            {name: 'width', width: 60, sorttype: "text"},
            {name: 'height', width: 60, sorttype: "text"},
            {name: 'numb', width: 60, sorttype: "text"},
            {name: 'angl1', width: 60, sorttype: "text"},
            {name: 'angl2', width: 60, sorttype: "text"}
        ]
    });
}

kits.load_table1 = function (table) {

}

File: src/java/model/Kits.java
package model;

import dataset.Query;
import dataset.Record;
import domain.eProkit;
import domain.eSysuser;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.HashMap;
import java.util.List;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import model.sys.App;
import model.sys.Att;
import org.json.simple.JSONObject;

public class Kits {

    public JSONObject kitsList(HttpServletRequest request, HttpServletResponse response) {
        ArrayList<List> list = new ArrayList();

        Query qProkit = new Query(Att.att(request).connect(), eProkit.values()).select(eProkit.up);
        for (Record rec : qProkit) {
            list.add(Arrays.asList(
                    rec.get(eProkit.id),
                    rec.get(eProkit.artikl_id),
                    rec.get(eProkit.artikl_id),
                    rec.get(eProkit.color1_id),
                    rec.get(eProkit.color2_id),
                    rec.get(eProkit.color3_id),
                    rec.get(eProkit.width),
                    rec.get(eProkit.height),
                    rec.get(eProkit.numb),
                    rec.get(eProkit.angl1),
                    rec.get(eProkit.angl2)));
        }
        JSONObject output = new JSONObject(App.asMap("kitsList", list));
        return output;
    }
}
